models/articles-models.js

```diff
@@ -9,3 +9,23 @@ exports.articleFromId = (article_id) => {
             return rows[0]
         });
 };
+
+exports.allArticles = ()=>{
+    return db.query(`SELECT articles.article_id,
+            articles.title,
+            articles.topic,
+            articles.author,
+            articles.created_at,
+            articles.votes,
+            articles.article_img_url,
+            COUNT(comments.article_id) AS comment_count FROM articles
+        LEFT JOIN 
+            comments ON articles.article_id = comments.article_id WHERE articles.votes >= 0 
+        GROUP BY 
+            articles.article_id 
+        ORDER BY 
+            articles.created_at DESC
+    `).then(({rows})=>{
+        return rows;
+    })
```

Comment:
Do you need to check that the article.votes are greater than or equal to 0? Shouldn't they all be anyway and even if they were minus why would you want to omit them? 

You can do a JOIN without a WHERE clause :)